### DISCUSSION TITLE
Media link is not being deleted fast enough

### DISCUSSION THREAD

**Discussion by MrNewbie72** [pixelfed/pixelfed#5122] [General]
When deleting a post i still see the link to the image in https://pixel.example.com/i/admin/media

Examples

![pixelfed4](https://user-images.githubusercontent.com/52122485/161899921-a5a9f37c-2693-44ef-a2a6-1ccd525e5519.png)

and i assume this is affecting the storage usage in my account settings.

![pixelfed5](https://user-images.githubusercontent.com/52122485/161900109-8f5c4c66-25c7-4457-b9ab-a3a034233511.png)

because i have uploaded and delete a lot of images/videos for testing but the storage count never went down.

**Comment by fofwisdom** [comment]
The problem is that we also preserve the spammer's media for a long period of time, during which time the spammer can abuse it

**Comment by fofwisdom** [comment]
Spammers have been using my Pixelfed image storage for ads for four months now. Maybe five months.
